stable/rocketchat/values.yaml

```diff
@@ -2,8 +2,8 @@
 ## ref: https://hub.docker.com/r/rocketchat/rocket.chat/tags
 ##
 image:
-  repository: docker.io/rocketchat/rocket.chat
```

Comment:
Why are you removing the `docker.io` part?
It's considered best practice to explicitly configure the registry url, since systems might be configured with another default than `docker.io`.

Reply:
i thought it was obsolete. Didn't know you can change the default registry url.